src/invidious/comments.cr

```diff
@@ -181,6 +181,8 @@ def fetch_youtube_comments(id, cursor, format, locale, thin_mode, region, sort_b
               json.field "content", html_to_content(content_html)
               json.field "contentHtml", content_html
 
+              json.field "isPinned", (node_comment["pinnedCommentBadge"]? != nil)
```

Comment:
We should probably check that `pinnedCommentBadge` is not equal to `false`, just in case.

Reply:
```suggestion
              json.field "isPinned", (node_comment["pinnedCommentBadge"]?.try(&.as_bool) == true)
```

Reply:
Sounds good

Reply:
My bad, the boolean check is causing issues because that element is actually not a boolean.
https://github.com/iv-org/invidious/pull/3631

Reply:
It's my fault too 😅, I thought `.try(&.as_bool)` would have returned the truthy/falsey value (similar to how JavaScript does it).